src/renderer/corporate/corporate-container.jsx

```diff
@@ -23,7 +23,10 @@ class CorporateContainerComponent extends PureComponent {
 					path={`${match.path}/setup-corporate-profile/:identityId`}
 					component={CorporateWizardContainer}
 				/>
-				<Route path={`${match.path}/add-member`} component={CorporateAddMemberContainer} />
+				<Route
+					path={`${match.path}/add-member/:parentId?`}
+					component={CorporateAddMemberContainer}
```

Comment:
better to use "render" instead of "component".  Component will cause component remount on each state render

Which in turn can be caused by unrelated state change.

Untill we nail memoized selectors correctly this might be a performance issue

Reply:
Fixed on https://github.com/SelfKeyFoundation/Identity-Wallet/pull/1832/commits/61ddf2356740d6d2822b51c131d08a3863272fed